src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const rentCar = 40;
```

Comment:
I guess it's daily pay for renting a car. It's not clear from variable's name.
Let's make it clear